### PR TITLE
CEDS-1954  New 'Inland mode of transport' page

### PR DESCRIPTION
**Pull request by janeturner**
Renamed Warehouse Details to Inland Transport Details in routes, Controller, view and tests.

### CODE REVIEW

**Comment by PatrykRudnicki** [comment]
Please run scala formatter

File: app/controllers/navigation/Navigator.scala [hunk 1]
```diff
@@ -69,7 +69,7 @@ object Navigator {
   }
 
   val simplified: PartialFunction[DeclarationPage, Mode => Call] = {
-    case BorderTransport        => controllers.declaration.routes.WarehouseDetailsController.displayPage
+    case BorderTransport        => controllers.declaration.routes.InlandTransportDetailsController.displayPage
```
Comment (PatrykRudnicki) [inline app/controllers/navigation/Navigator.scala]: Is this page not for all journeys? Is it just for Simplified?

Comment (PatrykRudnicki) [reply app/controllers/navigation/Navigator.scala]: Not page, I mean this redirect :)

**Comment by PauloPortugal** [comment]
@bambuchaAdm Acceptance tests here: https://github.com/hmrc/cds-exports-acceptance/pull/161

File: app/controllers/navigation/Navigator.scala (hunk 1 above)
Comment (PauloPortugal) [reply app/controllers/navigation/Navigator.scala]: It is for all journeys

**Comment by bambuchaAdm** [comment]
@PauloPortugal Nice - so now only blocker is tempermonkey

**Pull request merged**